scapy/contrib/tzsp.uts

```diff
@@ -0,0 +1,503 @@
+from enum import test
```

Comment:
Are you sure that this is needed ? What is it used for ?

Reply:
not used at all - removed.